### PR TITLE
Feature/nsfw nft

### PR DESCRIPTION
**Pull request by CrazyOptimist**
##### Done
- [x] Add hentai and porn metrics (from 0 to 100) to nfts data
- [x] Add sliders to user profile for selecting 0-100 value to filter images
- [x] Blur images with higher values if hiding NSFW is turned on in settings, it is on by default

##### Note
No designs for this currently, so please check only logic

##### Screenshot(s)
image

### CODE REVIEW

File: src/common/components/NFTCard/NFTCard.tsx [hunk 1]
```diff
@@ -159,20 +163,23 @@ const NFTCard = ({
...
         </div>
       ) : null}
       <Link to={detailUrl} className='cursor-pointer w-full'>
         <div
           className={cn(
-            'relative',
+            'relative overflow-hidden',
             isPortfolio && 'h-220px',
             !isPortfolio && imageHeightClass,
           )}
```
Comment (sanjanadhikari1) [inline src/common/components/NFTCard/NFTCard.tsx:175]: I think it's better to use isPortfolio? 'h-220px': imageHeightClass. Can we change this?

**Review by sanjanadhikari1 — CHANGES_REQUESTED** [review]
Others looks good to me

File: src/common/components/NFTCard/NFTCard.tsx (hunk 1 above)
Comment (CrazyOptimist) [reply src/common/components/NFTCard/NFTCard.tsx:175]: Fixed, though I don't think it is better and this wasn't changed by this PR

Comment (sanjanadhikari1) [reply src/common/components/NFTCard/NFTCard.tsx]: okay I can see this is not belongs to PR. Anyway thank you!

**Review by sanjanadhikari1 — APPROVED** [review]

**Review by ngvtuan — APPROVED** [review]

**Pull request merged**